src/use-cases/getCurrentUser.ts

```diff
@@ -0,0 +1,6 @@
+import { Request, Response } from 'express';
+import currentUser from '../mock-data/currentUser.json';
```

Comment:
We've just merged in the configurable data stuff. If you merge master into this and do something like 
```
import getMockData from '../getMockData';

export const getCurrentUser = () => (req: Request, res: Response): void => {
    const currentUser = getMockData('currentUser.json');
    res.json(currentUser);
};

```